### ISSUE TITLE
Rule book link near the download game button

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea-game.github.io#45]
Task: place a link to the rule book somewhere near the download game button:

https://github.com/triplea-game/assets/tree/master/rulebooks

**Comment by RoiEXLab** [comment]
@DanVanAtta this can br closed as well

**Comment by DanVanAtta** [comment]
@RoiEXLab  - there was an error in the #58 
```
The page build failed with the following error:

You have an error on line 2 of your `_config.yml` file. For more information, see https://help.github.com/articles/page-build-failed-config-file-error.
```

**Comment by RoiEXLab** [comment]
@DanVanAtta Fixed by #61
